homeassistant/components/satel_integra/__init__.py

```diff
@@ -25,12 +26,14 @@
 
 CONF_DEVICE_HOST = 'host'
 CONF_DEVICE_PORT = 'port'
-CONF_DEVICE_PARTITION = 'partition'
+CONF_DEVICE_CODE = 'code'
+CONF_DEVICE_PARTITIONS = 'partitions'
```

Comment:
You are renaming `partition` to `partitions`, that looks like a breaking change?

Reply:
Correct, this is a breaking change. Do I need to do anything special about it?

Reply:
Please add a paragraph in the PR description about the breaking change for the release notes. Write it from the point of view of the user, so that the user knows what to do to handle the breaking change.

Reply:
I'll add breaking change label.

Reply:
OK, added paragraph in the description with sample fix for the config.